Server/ConcordiaCurriculumManager/appsettings.Development.json

```diff
@@ -21,15 +21,34 @@
   "SenderEmailSettings": {
     "SenderSMTPHost": "smtp.gmail.com",
     "SenderSMTPPort": "587",
-    "SenderEmail": "test@gmail.com",
-    "SenderPassword": "not a valid password"
+    "SenderEmail": "ccmc9531@gmail.com",
+    "SenderPassword": "fgym vpwn eths umii"
```

Comment:
If this email is linked to your phone number, you may want to remove this from version control. It's up to you.

Reply:
Ok, ill take it off